src/mongocxx/read_preference.hpp

```diff
@@ -238,6 +238,28 @@ class MONGOCXX_API read_preference {
     ///
     stdx::optional<std::chrono::seconds> max_staleness() const;
 
+    /// Sets the hedge document to be used for the read preference. Sharded clusters running MongoDB
```

Comment:
nit, add empty /// comment lines above this first text line and below the last text line in this comment.

Reply:
Done. That is consistent with other comment blocks.